store/actions/user.js

```diff
@@ -10,7 +10,7 @@ import Router from 'next/router'
 
 // Component imports
 import actionTypes from '../actionTypes'
-
+import initialState from '../initialState'
```

Comment:
is it proper to do this within redux? I'm not sure XD

Reply:
Seems like a solid solution to me.